src/Game/zMusic.cpp

```diff
@@ -7,104 +7,111 @@
 extern zMusicTrackInfo sMusicTrack[2];
 extern zVolumeInfo volume;
 extern uint32 sMusicPaused;
-extern LevelMusic sMusicLastEnum[2];
+extern int32 sMusicLastEnum[2];
 extern float32 unkFloat; //0f by default.
 extern float32 minDelay; //Value is defaulted at 0.001f.
 
 // func_800A6E9C
 #pragma GLOBAL_ASM("asm/Game/zMusic.s", "volume_reset__Fv")
 
-//Reset both music tracks to their default volume.
+// Reset both music tracks to their default volume.
 void zMusicRefreshVolume()
 {
-  	for (int32 i = 0; i < 2; i++)
-  	{
-  		zMusicTrackInfo* track = &sMusicTrack[i];
-  		if (track->snd_id != 0)
-  		{
-  			xSndSetVol(track->snd_id, volume.cur); //Only reset volume if the sound ID is not blank.
-  		}
-  	}
+    for (int32 i = 0; i < 2; i++)
+    {
+        zMusicTrackInfo* track = &sMusicTrack[i];
+        if (track->snd_id != 0)
+        {
+            xSndSetVol(track->snd_id,
+                       volume.cur); // Only reset volume if the sound ID is not blank.
```

Comment:
Nitpicking, but I'd put this comment above the function call instead of after to keep the function call on a single line